src/datepicker/datepicker.js

```diff
@@ -150,7 +151,9 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.position'])
       }
 
       // Configuration parameters
-      var mode = 0, selected = new Date(), showWeeks = datepickerConfig.showWeeks;
+      var mode = 0;
+      var selected = scope.focusedDate = new Date();
```

Comment:
This means `focusedDate` will always be overriden, so why provide it in the `scope` definition?